packages/forms/frontend/sirius-components-forms/src/groups/Group.tsx

```diff
@@ -53,14 +53,19 @@ const useGroupStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
   verticalSections: {
     display: 'flex',
     flexDirection: 'column',
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : null),
+    alignItems: ({ borderStyle }) => borderStyle?.alignItems || null,
+    justifyContent: ({ borderStyle }) => borderStyle?.justifyContent || null,
     '& > *': {
       marginBottom: theme.spacing(2),
     },
   },
   adaptableSections: {
     display: 'grid',
     gridTemplateColumns: 'repeat(auto-fit, minmax(250px, 1fr))',
-    gap: theme.spacing(2),
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : theme.spacing(2)),
```

Comment:
Same comment regarding default values AND the grid algorithm. On top of that, the value of `alignItems` and `justifyContent` are not the same in both algorithm.

Reply:
Customization is for flexbox only and the properties are not link to border anymore